.github/actions/prep-release/action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: "If set, do not make a PR"
     default: "false"
     required: false
+  silent:
+    description: "Set a placeholder in the changelog and don't publish the release."
```

Comment:
Should the docs be updated to give more information about this new input? Since it will show up in the workflow trigger popup.

Reply:
Especially since most people will never use it.